src/code/z_player_lib.c

```diff
@@ -1061,7 +1061,7 @@ s32 Player_OverrideLimbDrawGameplayCommon(PlayState* play, s32 limbIndex, Gfx**
         // Note: The increment would not be done for the root limb, even if it had a non-NULL `dList`.
         //       So if the root limb had a non-NULL `dList` (which is not the case in vanilla),
         //       an out-of-bounds write to `bodyPartsPos` would occur.
-        sCurBodyPartPos = &this->bodyPartsPos[-1];
+        sCurBodyPartPos = &this->bodyPartsPos[0] - 1;
```

Comment:
I can't believe gcc is fooled by this one lmao

Reply:
I guess it makes more sense like this since the later operation is `sCurBodyPartPos++` but yeah it's weird only one of the two is a warning when afaik both writings are exactly the same per the standard

Reply:
I prefer the way it was before, but if its for the sake of the warning i guess its fine. I just recognize way quicker what happening with a -1 index instead of having to think about what a -1 after the fact does